auth0/src/main/java/com/auth0/android/authentication/ParameterBuilder.kt

```diff
@@ -64,7 +66,7 @@ public class ParameterBuilder private constructor(parameters: Map<String, String
      * @return itself
      */
     public fun setScope(scope: String): ParameterBuilder {
-        return set(SCOPE_KEY, scope)
+        return set(SCOPE_KEY, OidcUtils.includeRequiredScope(scope))
```

Comment:
when a scope is set through this setter, it will be checked so that it contains "openid"